pkg/api/v1/conversion.go

```diff
@@ -373,63 +372,14 @@ func Convert_api_PodStatusResult_To_v1_PodStatusResult(in *api.PodStatusResult,
 		return err
 	}
 
```

Comment:
Run `hack/update-generated-docs.sh` smells work. Don't need to remove `Convert_v1_ReplicationControllerSpec_To_api_ReplicationControllerSpec`. Right?